jar-infer/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -90,9 +95,19 @@
    * Check set equality of results with expected results
    *
    */
-  public boolean verify(Set<String> result, Set<String> expected) {
-    for (String var : result) {
-      if (!expected.remove(var)) return false;
+  public boolean verify(
```

Comment:
Should this be part of the library code itself, or just a helper method of the testing infrastructure?